### DISCUSSION TITLE
Update reorder site awards

### DISCUSSION THREAD

**Discussion by EldridgeCrimson** [RetroAchievements/RAWeb#1068] [Ideas]
Hello everyone,

I think that a beneficial change to reordering the site awards would be to have a UI where you can drag and drop the awards and see what the section will look like before you submit it. Currently, you just type a number in whichever order you want the award to be presented in, but with my suggestion, I believe it will be easier to customize your awards and have more personalization on your page. I am curious to everyone's thoughts on this.

EldridgeCrimson

**Comment by Ryudo300** [comment]
Maybe a grid view as well to go along with this. Particularly handy when you have a set that has a lot of achievements, so everything is visible from the get go, rather than having to scroll through a giant list.

**Reply by EldridgeCrimson** [reply]
I should have clarified better in my original post, but yes this is exactly what I had in mind as well. Thank you for suggesting it

**Comment by luchaos** [comment]
As for any type of reordering on the site it would be great to have drag&drop in place. 
Should be very doable in the future when the new frontend stack is in place.

It should be possible to have both list and grid views, as mentioned, some resources tend to come in large amounts.

See https://completionist.me/retro/profile/luchaos/game/10003/kanban for reference.

**Reply by EldridgeCrimson** [reply]
I myself do not see a use for having a list view in the things that I do, but since you have recommended it, I'm sure there are others who feel the same, and this would be great to have as well.
